server/src/main/java/io/seata/server/storage/redis/LuaParser.java

```diff
@@ -149,4 +157,14 @@ public static <T> List<T> getListFromJson(String json, Class<T> classz) {
             throw new StoreException(e.getMessage());
         }
     }
+
+    public static Object jedisEvalSha(Jedis jedis, String luaSHA, String luaFileName, List<String> keys, List<String> args) {
+        try {
+            return jedis.evalsha(luaSHA, keys, args);
+        } catch (JedisNoScriptException e) {
```

Comment:
Can this handling method be applied to other Redis exceptions as well, not just limited to JedisNoScriptException?

Reply:
other redis exceptions do not require script load